src/content/docs/ai-observability/intro-to-ai-monitoring.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: 'Introduction to AI Monitoring'
+metaDescription: 'AI Monitoring lets you observe the AI-layer of your tech stack, giving you a holistic overview of the health and performance of your AI-powered app.'
+freshnessValidatedDate: never
+---
+
+import apmAiMonitoringAllCapabilities from 'images/apm_screenshot-full_ai-monitoring-all-capabilities.gif'
+
+Your AI-powered app introduces new technologies into your environment that are hard to monitor, let alone understand. With AI Monitoring, you can bridge unfamiliar technologies to familiar solutions, deepening your knowledge about your AI's behavior so you can improve its performance. 
+
+<img
+    title="View your app's AI data"
+    alt="A gif that shows where to find your AI data in New Relic"
+    src={apmAiMonitoringAllCapabilities}
+/>
+
+<figcaption>
+    Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI Monitoring. Choose your entity from the AI entities page to view your data.
+</figcaption>
+
+* **Evaluate errors and bugs on the code-level.** Get total insight over how your AI processes perform and make improvements when they fail. 
```

Comment:
"on the code-level" feels odd to me, but I'm having trouble putting my finger on why.

Why not something like "Evaluate errors and bugs in your AI code"?